base/docs/basedocs.jl

```diff
@@ -2515,8 +2515,8 @@ kw"Union{}", Base.Bottom
 """
     Union{Types...}
 
-A type union is an abstract type which includes all instances of any of its argument types. The empty
-union [`Union{}`](@ref) is the bottom type of Julia.
+A `Union` type is an `abstract type` that can reference any value of one of its arguments. It cannot be
```

Comment:
I like the wording you propose, it makes it clear what is the relation with the type parameters.

On the fact that it cannot be instantiated, coming from languages as C/C++ one maybe think that it _could_ be instantiated, as it is possible on those. So for user-friendliness I would prefer to keep it there.

Reply:
My issue was specifically with the phrase "that can reference". The word reference has a common (and unrelated) meaning in programming, so I think it should be avoided here. The phrase "is a supertype of" is what I would prefer, since that is the most accurate description in this context.

As for instantiation, my point is about abstract types in general. C++ doesn't have those, and it is important for any julia user to learn that an "abstract type" is not the same as a "parent class", and to not think of these terms as related. I think conflating or associating those two ideas in individual docstrings only leads to more confusion in the long-run.

Reply:
> The word reference has a common (and unrelated) meaning in programming, so I think it should be avoided here.

I agree with you on this. Maybe to say  'hold' instead of 'reference'?

Reply:
I don't think abstract types (or unions) "hold" anything, in any meaningful sense. In my opinion, the only *totally correct* thing that can be said here is in terms of type relations. 

Maybe only a slight change to the original is needed, like:
```
A type union is an abstract type which includes all instances of any of its parameter types. I.e. for any types `T` and `S`, `T <: Union{T, S}` and `S <: Union{T, S}`
```

Maybe it's more helpful to beginners to just point them to the documentation on types 🤷‍♂️